react-ui/src/sprint/index.js

```diff
@@ -23,83 +24,88 @@ const SprintDetail = ({ sprintDetails, attributes, token, projectName }) => {
+                method: "post",
+                data: formData,
+                headers: {
+                    "token": token,
+                    "Content-Type": "application/json",
+                    "Access-Control-Allow-Origin": "http://localhost:3000/project"
+                }
+            })
+            .then(res => ({ [option.value]: res.data }))
+            .catch(ex => {
+                console.error(ex);
+                setError(true);
+            });
+        });
 
-        axios({
-            url: "http://localhost:8000/metric/Burndown",
-            method: "post",
-            data: formData,
-            headers: {
-                "token": token,
-                "Content-Type": "application/json",
-                "Access-Control-Allow-Origin": "http://localhost:3000/project"
-            }
-        })
-        .then(res => {
-            setData(res.data);
-            console.log(res.data);
-            setError(false);
-            setSpinner(false);
-        })
-        .catch(ex => {
-            console.log(ex);
-            setError(true);
+        Promise.all(fetchDataForSelectedValues).then(results => {
+            const newData = results.reduce((acc, result) => ({
+                ...acc,
+                ...result
+            }), {});
+
+            setData(newData);
             setSpinner(false);
         });
-    }
+    };
+
+    const options = sprintDetails.map(detail => ({
+        value: detail.id,
+        label: detail.name
+    }));
 
     return (
         <div>
-            <div>
-                <Stack gap={4}>
-                    <h3 className="projectName">{projectName}</h3>
-                    <div className="d-flex justify-content-center backgroundWhite">
-                        <br />
-                        <Dropdown onSelect={handleSelect}>
-                            <Dropdown.Toggle variant="outline-secondary" className="backgroundButton">
-                                {selectedValue ? selectedValue : 'Select Metric'}
-                            </Dropdown.Toggle>
-                            <Dropdown.Menu>
-                                {
-                                    sprintDetails.map((item) => <Dropdown.Item key={item.id} eventKey={[item.id, item.name]}>{item.name}</Dropdown.Item>)
-                                }
-                            </Dropdown.Menu>
-                        </Dropdown>
-                    </div>
+            <Stack gap={4}>
+                <h3 className="projectName">{projectName}</h3>
+                <div className="d-flex justify-content-center backgroundWhite">
+                    <Select
+                        isMulti
+                        options={options}
+                        className="basic-multi-select"
+                        classNamePrefix="select"
+                        onChange={handleSelect}
+                        placeholder="Select Metrics"
```

Comment:
Could you please change the placeholder to "Select Sprints" instead?

Reply:
Modified